### PR TITLE
Here is three commits to solve #5 issue 

### PR DESCRIPTION
**Pull request by ramusus**
- made migration compatible with Django 1.7, added index on field_name column
- ability to change serialize name in settings: FIELD_HISTORY_SERIALIZER_NAME
- added json_nested_serializer for serializing fields from parent model

### CODE REVIEW

File: field_history/migrations/0001_initial.py [hunk 1]
```diff
@@ -12,7 +12,6 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('contenttypes', '0002_remove_content_type_name'),
```
Comment (grantmcconnaughey) [inline field_history/migrations/0001_initial.py]: Why was this removed?

**Comment by grantmcconnaughey** [comment]
Nice, lots of improvements here it looks like.

A few things I'd like to see before merging this:
* Tests. This took test coverage from 95% to 62%. Let's get that `json_nested_serializer.py` file completely tested.
* Documentation. Add a changelog entry for version 0.5.0 with the changes and put the `FIELD_HISTORY_SERIALIZER_NAME` setting in the documentation somewhere.

I need to look over it a bit more before giving more feedback, but those two things are a start.

File: field_history/migrations/0001_initial.py (hunk 1 above)
Comment (ramusus) [inline field_history/migrations/0001_initial.py]: it's for consistency with Django 1.7.5. At that time there was no such migration

**Comment by ramusus** [comment]
Ok, agree will update PR

File: field_history/migrations/0001_initial.py (hunk 1 above)
Comment (ramusus) [inline field_history/migrations/0001_initial.py]: and mention of this migration does not make sense here

Comment (grantmcconnaughey) [inline field_history/migrations/0001_initial.py]: The `FieldHistory` model has a `ContentType` field, so we'll need to make sure that the migrations for `contenttypes` have completed before creating the `FieldHistory` table or else this migration will fail.

**Comment by grantmcconnaughey** [comment]
Also, if you want to support Django 1.7.x then put that in `.travis.yml` so I can confirm all tests pass for it. [Support for Django 1.7 ended on December 1, 2015](https://www.djangoproject.com/download/), so I'm not going to do anything to add support for it. But, if you can get it working and it doesn't muddy up the codebase then I will accept the PR. Just know that if I have some feature I really want to add and it is Django 1.8+ only, then it will take precedence over keeping Django 1.7 compatibility.

File: field_history/migrations/0001_initial.py (hunk 1 above)
Comment (ramusus) [inline field_history/migrations/0001_initial.py]: You are right, I will return it back

**Comment by ramusus** [comment]
@grantmcconnaughey I fixed coveralls and travis issues, added tests. I didn't update documentation, because this functionality should be supported by default, I suppose. 
Check, please is it ready to merge

**Comment by grantmcconnaughey** [comment]
Awesome stuff, @ramusus. Give me the weekend to take a final look and write up some release notes.

**Comment by grantmcconnaughey** [comment]
django-field-history 0.5.0 has been deployed. Thanks for the work!
